Bhagyashri/Correction/css/style.css

```diff
@@ -0,0 +1,109 @@
+
+body{
+background-color:#006966;
+padding-left:100px;
+padding-right:100px;
+margin:0px;
```

Comment:
add padding: 0 100px;
margin: 0;
whenever we are using 0 we don't need unit.

Reply:
okay